registrations/tasks.py

```diff
@@ -380,7 +373,13 @@ def run(self, registration_id, **kwargs):
                 self.set_risk_status(registration)
 
             self.l.info("Scheduling registration push to Jembi")
-            self.send_to_jembi(registration)
+            jembi_task = BasePushRegistrationToJembi.\
+                get_jembi_task_for_registration(registration)
+            task = chain(
+                jembi_task.si(str(registration.pk)),
+                remove_personally_identifiable_fields.si(str(registration.pk))
```

Comment:
Yup, that was my understanding of `chain` from the celery docs. http://docs.celeryproject.org/en/latest/userguide/canvas.html#chains